cmake/ImportBSON.cmake

```diff
@@ -142,6 +142,16 @@ function (_import_bson)
       set (ENABLE_EXTRA_ALIGNMENT ${_extra_alignment_default} CACHE BOOL "Toggle extra alignment of bson_t")
       # We don't want the subproject to find libmongocrypt
       set (ENABLE_CLIENT_SIDE_ENCRYPTION OFF CACHE BOOL "Disable client-side encryption for the libmongoc subproject")
+      if (BUILD_VERSION)
+         # Both libmongocrypt and C driver support setting a `BUILD_VERSION`. Do not reuse the `BUILD_VERSION` for libmongocrypt when building the C driver.
+         # Set the `BUILD_VERSION` in this file to only apply to the C driver subproject.
+         set (BUILD_VERSION ${MONGOC_FETCH_TAG_FOR_LIBBSON})
+      endif()
```

Comment:
Is there a reason to only set the build version if its already defined for the parent project? This could also be `unset(BUILD_VERSION)` to clear the value from the scope and let the sub-project infer its own value, or alternatively it can unconditionally set `BUILD_VERSION` to the fetch tag value.

Reply:
No. Updated to unconditionally set `BUILD_VERSION`.